src/objects.c

```diff
@@ -1548,7 +1549,22 @@ s32 func_800210CC(s8 arg0) {
 
 GLOBAL_ASM("asm/non_matchings/objects/func_80021104.s")
 GLOBAL_ASM("asm/non_matchings/objects/func_8002125C.s")
-GLOBAL_ASM("asm/non_matchings/objects/func_80021400.s")
+
+void func_80021400(s32 arg0) {
+    s32 i;
+    arg0 &= 0xFF; //?
+
```

Comment:
```suggestion
    arg0 &= 0xFF;
```

I also assume that the function doesn't match if it's a u8/s8?

Reply:
Yeah, it's weird. If I change it to a u8 / s8 then the diff looks like this:
![image](https://user-images.githubusercontent.com/828639/187815465-311fe799-b31c-4680-a296-5a4fdce0828e.png)